protocols/v2/binary-sv2/no-serde-sv2/codec/src/datatypes/non_copy_data_types/seq_inner.rs

```diff
@@ -95,6 +95,15 @@ impl<'a, T: GetSize> GetSize for Seq0255<'a, T> {
     }
 }
 
+/// Fixed size data sequence up to a length of 65535
+///
+/// Byte Length Calculation:
```

Comment:
what is this referring to? where this byte calculation is happening?

Reply:
this is how is defined in spec. Actual byte calculation happen in binary-sv2.

Reply:
```rust
/// Return the encoded byte size of an `Encodable` comprehensive of the header, if any
pub trait GetSize {
    fn get_size(&self) -> usize;
}
```

Reply:
So should this doc live here or in the binary-sv2 and link to it?
another question please,  what does `L` refer to?

Reply:
Which `L` ?

Reply:
a ok the one in docs

Reply:
so that doc come directly from the spec and L stand for the number of elments in the sequnce
https://github.com/stratum-mining/sv2-spec/blob/main/03-Protocol-Overview.md#31-data-types-mapping

Reply:
yea, its mentioned here twice `Decsription: 2-byte length L, unsigned little-endian integer 16-bits, followed by a sequence of L elements of type T. Allowed range of length is 0 to 65535.`

Reply:
btw yes we could link the the GetSize and SizeHint trait in that doc. GetSize is implemented for Encodable and return the total len SizeHint is implemented for Decodable and retutrn total len - header len